isyfact-standards-doc/src/docs/antora/modules/einstieg/partials/Produktkatalog_IsyFact_Produktvorgaben_Bibliotheken_Anwendungsentwicklung_Java_Spring_Entwicklung.csv

```diff
@@ -16,15 +16,15 @@ Anwendungs-framework;https://spring.io/projects/spring-boot[Spring Boot];2.7.x;3
 "Anwendungs-framework +
 Test-Framework";https://spring.io/projects/spring-framework[Spring];<<spring-boot-dependencies,icon:leaf[]>>;2.4.0;verbindlich;Apache Software License;"* Die Kategorie Test-Framework bezieht sich auf die Features von Spring zur Testunterstützung.
 * Die folgenden Bestandteile des Spring Frameworks sind durch Architektur-entscheidungen explizit zur Nutzung freigegeben."
-Service-Frameworks;https://docs.spring.io/spring-framework/docs/current/reference/html/integration.html#remoting-httpinvoker[Spring Http Invoker];<<spring-boot-dependencies,icon:leaf[]>>;2.4.0;bestandsgeschützt;Apache Software License;
```

Comment:
Bitte in allen geänderten Zeilen die Spalte nach der geänderten Version auf "3.0.1" setzen. Diese Spalte steht dafür, in welcher Version der IsyFact die Versionsänderung geschehen ist.